content/using-sagemaker-managed-spot-training/notebooks/access_jupyterlab.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Accessing JupyterLab"
+chapter: false
+weight: 20
+---
+
+Now that you've deployed the CloudFormation template, you will be able to access an Amazon SageMaker Notebook Instance. An Amazon SageMaker Notebook Instance is a managed EC2 instance that comes pre-installed common tools used by data scientists to develop and execute data science experiments from Python Notebooks. 
+
+1. Browse to the Amazon SageMaker Console and locate the Amazon SageMaker Notebook Instance that was created by the CloudFormation template: https://us-west-2.console.aws.amazon.com/sagemaker/home?region=us-west-2#/notebook-instances ![SageMaker Notebook Instance](/images/using-sagemaker-managed-spot-training/jupyter-1.png)
```

Comment:
this needs to be a region-less link, so the user will be taken to the region where he's currently working in and has deployed the CFn stack in. I'm having trouble deep linking into the Notebook instances page, but even if the user gets to the dashboard in his region and clicks Notebook instances in the left pane I believe it's better than opening a static region link
https://console.aws.amazon.com/sagemaker/home?/notebook-instances/notebook-instances